packages/discord.js/src/structures/SectionComponent.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Component = require('./Component');
+const { createComponent } = require('../util/Components');
+
+/**
+ * Represents a section component
+ * @extends {Component}
+ */
+class SectionComponent extends Component {
+  constructor({ accessory, components, ...data }) {
```

Comment:
```suggestion
  constructor({ accessory, components = [], ...data }) {
```

Reply:
`components` is not optional in `APISectionComponent`, so why would this default be needed?